addons/fluxcd/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluxcd
-version: 2.2.0
+version: 2.2.1
```

Comment:
Thanks for your contribution! Given the size of this pull request, let's bump the addon version to 2.3.0.

Reply:
Yes upgraded to 2.3.0